elasticsearch/variables.tf

```diff
@@ -0,0 +1,169 @@
+variable "project" {
+  type        = string
+  description = "Project name"
+}
+
+variable "environment" {
+  type        = string
+  description = "Environment name"
+}
+
+variable "name" {
+  type        = string
+  description = "Name to use for the Elasticsearch domain"
+}
+
+variable "elasticsearch_version" {
+  type        = string
+  description = "Version of the Elasticsearch domain"
+  default     = "6.7"
+}
+
+variable "options_rest_action_multi_allow_explicit_index" {
+  type        = bool
+  description = "Sets the `rest.action.multi.allow_explicit_index` advanced option. When set to `false`, Elasticsearch will reject requests that have an explicit index specified in the request body"
+  default     = true
+}
+
+variable "options_indices_fielddata_cache_size" {
+  type        = string
```

Comment:
You set this to `number`  and `null` as default in the readme 🙂

Reply:
Right